src/testWithSpringSecurity_5_8/java/org/openrewrite/spring/security5/RemoveUseAuthorizationManagerTest.java

```diff
@@ -0,0 +1,62 @@
+package org.openrewrite.spring.security5;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.config.Environment;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.xml.Assertions.xml;
+
+public class RemoveUseAuthorizationManagerTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipe(Environment.builder()
+          .scanRuntimeClasspath()
+          .build()
+          .activateRecipes("org.openrewrite.java.spring.security6.RemoveUseAuthorizationManager"));
+    }
+
+    @Test
+    void removeUseAuthorizationManagerAttribute() {
```

Comment:
Is this test necessary? Seems like it mostly tests `ChangeTagAttribute`, which already has its own tests.

Reply:
Yes, `ChangeTagAttribute` didn't support removing an attribute before and I just changed it to support it.
I think adding an example to demonstrate this specific use case is no harm.